tests/Storage.test.ts

```diff
@@ -520,6 +523,35 @@ function putAllTransactionData (ledger_storage: LedgerStorage,
     doPut();
 }
 
+/**
+ * Run Validators API tests
+ */
+function runValidatorsAPITest (ledger_storage: LedgerStorage, callback?: (err: Error | null, rows: any[]) => void)
+{
+  var address: string = 'GA3DMXTREDC4AIUTHRFIXCKWKF7BDIXRWM2KLV74OPK2OKDM2VJ235GN';
+  var res = ledger_storage.getValidatorsAPI(1, null, (err1: Error | null, rows: any[]) =>
+  {
+      assert.ok(!err1, err1?.message);
+      assert.equal(rows[0].address, address);
+      assert.equal(rows[0].enrolled_at, 0);
+      assert.equal(rows[0].distance, 0);
+
+      var res1 = ledger_storage.getValidatorsAPI(1, address, (err2: Error | null, rows: any[]) =>
+      {
+          assert.ok(!err2, err2?.message);
+          assert.equal(rows.length, 1);
+          assert.equal(rows[0].address, address);
+
+          var res2 = ledger_storage.getValidatorsAPI(null, null, (err3: Error | null, rows: any[]) =>
+          {
+              assert.ok(!err3, err3?.message);
+              assert.equal(rows.length, 3);
+              assert.equal(rows[0].distance, 0);
+          });
```

Comment:
Callback should be called after the test as there may be other tests.
```suggestion
              assert.equal(rows[0].distance, 0);
              callback();
          });
```